swagger/swagger.json

```diff
@@ -267,29 +267,29 @@
           "example" : "your-reference",
           "readOnly" : true
         },
-        "created_date" : {
+        "description" : {
```

Comment:
I cannot see you've done any changes to Documentation/Examples. Are these intentional changes?